cli/tests/badly_formatted.js

```diff
@@ -1,4 +1,4 @@
 
-console.log(
-  "Hello World"
+console.log("Hello World"
+
 )
```

Comment:
Undo this change

Reply:
It's on purpose, `dprint` keeps string on separate line if you line break after parentheses...

Reply:
Opened https://github.com/dsherret/dprint/issues/61